receiver/prometheusreceiver/metrics_receiver_helper_test.go

```diff
@@ -447,6 +447,9 @@ func compareAttributes(attributes map[string]string) numberPointComparator {
 		if req {
 			for k, v := range attributes {
 				val, ok := numberDataPoint.Attributes().Get(k)
+				if !ok {
```

Comment:
I think you can just give `require.True` a message instead of checking the false condition here. https://pkg.go.dev/github.com/stretchr/testify/require#True

Reply:
ughghghghg my bad added this here so I could test using breakpoints, forgot to remove this